w3-hw1-ThuyDung/cypress/pages/registrationForm/formPage.js

```diff
@@ -77,14 +57,7 @@ export const formPage ={
     },
     checkHobbies(hobbies){
         for(let i=0; i< hobbies.length; i++){
```

Comment:
You can use `forEach`, `for(i=0)` is usually used when you need the index to do something